src/registrar/templates/django/admin/domain_application_change_form.html

```diff
@@ -0,0 +1,96 @@
+    {% endcomment %}
+    {# Create a modal for when a domain is marked as ineligible #}
+    <div
+    class="usa-modal"
+    id="toggle-set-ineligible"
+    aria-labelledby="Are you sure you want to select ineligible status?"
+    aria-describedby="This application will be marked as ineligible."
+    >
+        <div class="usa-modal__content">
+            <div class="usa-modal__main">
+                <h2 class="usa-modal__heading" id="modal-1-heading">
+                    Are you sure you want to select ineligible status?
+                </h2>
+                <div class="usa-prose">
+                    <p>
+                        When a domain request is in ineligible status, the registrant's permissions within the registrar are restricted as follows: 
+                    </p>
+                    <ul>
+                        <li class="font-body-sm">They cannot edit the ineligible request or any other pending requests.</li>
+                        <li class="font-body-sm">They cannot manage any of their approved domains.</li>
+                        <li class="font-body-sm">They cannot initiate a new domain request.</li>
+                    </ul>
+                    <p>
+                        The restrictions will not take effect until you “save” the changes for this domain request. 
+                        This action can be reversed, if needed.
+                    </p>
+                    <p>
+                        Domain: <b>{{ original.requested_domain.name }}</b>
+                        {# Acts as a <br> #}
+                        <div class="display-inline"></div>
+                        New status: <b>{{ original.ApplicationStatus.INELIGIBLE|capfirst }}</b>
+                    </p>
+                </div>
+
+                <div class="usa-modal__footer">
+                    <ul class="usa-button-group">
+                        <li class="usa-button-group__item">
+                            <button
+                            type="submit" 
+                            class="usa-button" 
+                            name="_set_application_ineligible"
+                            data-close-modal
+                            >
+                                Yes, select ineligible status
+                            </button>
+                        </li>
+                        <li class="usa-button-group__item">
+                            <button
+                            type="button"
+                            class="usa-button usa-button--unstyled padding-105 text-center"
+                            name="_cancel_application_ineligible"
+                            data-close-modal
+                            >
+                                Cancel
+                            </button>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <button
```

Comment:
Should we also connect the same logic used for "_cancel_application_ineligible" to this button + the event the user clicks outside of the modal?